hr_attendance_report_theoretical_time/README.rst

```diff
@@ -0,0 +1,167 @@
+=====================================
+Theoretical vs Attended Time Analysis
+=====================================
+
+.. !!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
+   !! This file is generated by oca-gen-addon-readme !!
+   !! changes will be overwritten.                   !!
+   !!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
+
+.. |badge1| image:: https://img.shields.io/badge/maturity-Beta-yellow.png
+    :target: https://odoo-community.org/page/development-status
+    :alt: Beta
+.. |badge2| image:: https://img.shields.io/badge/licence-AGPL--3-blue.png
+    :target: http://www.gnu.org/licenses/agpl-3.0-standalone.html
+    :alt: License: AGPL-3
+.. |badge3| image:: https://img.shields.io/badge/github-OCA%2Fhr-lightgray.png?logo=github
+    :target: https://github.com/OCA/hr/tree/13.0/hr_attendance_report_theoretical_time
+    :alt: OCA/hr
+.. |badge4| image:: https://img.shields.io/badge/weblate-Translate%20me-F47D42.png
+    :target: https://translation.odoo-community.org/projects/hr-12-0/hr-12-0-hr_attendance_report_theoretical_time
```

Comment:
There is still something referring to branch 12.0 in this line: .......hr-12-0/hr-12-0-hr_attendance_report_theoretical_time
Did you generate the readme using maintainer-tools module?

Reply:
Hi 
![image](https://user-images.githubusercontent.com/55989612/71572150-1b754d00-2ade-11ea-92e7-f46a87ab2c13.png)

As far as I understand it I don't have to re-generate this file as it is automatically generated. @pedrobaeza am I wrong?

Reply:
It reduces automatic commits noise though

Reply:
README.rst file generated with "oca-gen-addon-readme" tool.

@pedrobaeza should this step of generating the README file be included in the [tasks todo in migration](https://github.com/OCA/maintainer-tools/wiki/Migration-to-version-13.0#tasks-to-do-in-the-migration)?

Reply:
> It reduces automatic commits noise though

@chienandalu How can I get automatically rid of these noise commits? As far as I know, this is a manual "squash" process described in the "tasks todo in migration" section

Reply:
@pegonzalezspesol it's not included in the steps of the migration guide because it's not mandatory. Indeed it's auto generated on merge, but pre-generating it reduces as said commit noise... and comments :stuck_out_tongue_closed_eyes: